### DISCUSSION TITLE
Editing Languages

### DISCUSSION THREAD

**Discussion by Midyy** [DexterHuang/CyberCodeOnline#2671] [Q&A]
Hello everyone!! My name is Mia and I'm new at Github.
Here's my question;
I've been editing the es.json, when i finish correcting i always click "save changes", but i'm not sure if it actualizes automatically or not, or if i should do a pull request. (╥﹏╥)

**Comment by fpoblete** [comment]
Hello. Are you native spanish speaker? Many of the last translations were terrible!

**Comment by Ravenant1234** [comment]
i have submitted a PR to improve translations

**Reply by Ravenant1234** [reply]
Done
